examples/nodejs/access-control/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "momento-nodejs-example",
+  "version": "1.0.0",
+  "description": "Getting started with Momento sdk",
+  "main": "index.ts",
+  "scripts": {
+    "prebuild": "eslint . --ext .ts",
+    "build": "tsc",
+    "access-control": "tsc && node dist/access-control.js",
+    "lint": "eslint . --ext .ts",
+    "format": "eslint . --ext .ts --fix"
+  },
+  "author": "",
+  "license": "ISC",
+  "devDependencies": {
+    "@types/node": "^16.11.4",
+    "@typescript-eslint/eslint-plugin": "5.30.5",
+    "@typescript-eslint/parser": "^5.0.0",
+    "eslint": "8.19.0",
+    "eslint-config-prettier": "8.5.0",
+    "eslint-plugin-import": "2.26.0",
+    "eslint-plugin-node": "11.1.0",
+    "eslint-plugin-prettier": "4.2.1",
+    "prettier": "2.7.1",
+    "typescript": "4.4.3"
+  },
+  "dependencies": {
+    "@gomomento/sdk": "file:../../../packages/client-sdk-nodejs"
```

Comment:
this is the right thing to do for this PR but we should come back and change it to a release version as soon as we have one.

We should also add this new example dir to CI, in the .github actions for PRs and for dependabot.

Those do not need to block this PR but we should do them when we can.

Reply:
Do we have a runbook for everything that needs to be done for releasing new SDKs? It would be really useful.

Reply:
this isn't really about new SDKs or releases, it's about new examples. we should add something in CONTRIBUTING.md.

Reply:
https://github.com/momentohq/client-sdk-javascript/pull/620